sdk/eventhub/event-hubs/test/internal/diagnostics/messageSpan.spec.ts

```diff
@@ -21,60 +19,24 @@ describe("#createMessageSpan()", () => {
     setTracer(origTracer);
   });
 
-  const mockSpanContext: SpanContext = {
-    traceId: "d4cda95b652f4a1592b449d5929fda1b",
-    spanId: "6e0c63257de34c92",
-    traceFlags: TraceFlags.SAMPLED
-  };
-  const mockEventHubConnectionConfig: Pick<EventHubConnectionConfig, "entityPath" | "host"> = {
-    entityPath: "entity",
-    host: "foo.example.com"
-  };
-
   it("should create a span without a parent", () => {
-    const span = createMessageSpan();
+    const { span } = createMessageSpan(
+      {},
+      {
+        entityPath: "entity path",
+        host: "host"
+      }
+    );
 
     should.exist(span);
     should.exist(span.context().spanId);
     should.exist(span.context().traceId);
 
-    should.equal((span as any).name, "Azure.EventHubs.message");
-    assert.deepStrictEqual((span as any).attributes, {
-      "az.namespace": "Microsoft.EventHub"
-    });
-
-    span.end();
-  });
-
-  it("should create a span with a parent", () => {
```

Comment:
Is there actually not a way to create a span with a parent context until you finish the upgrade then?

Reply:
There is, you just fill out `options.tracingOptions` when you call createSpan. 

I removed this test because it seemed redundant with testing that already happens (in core-tracing's tests) for parent context propagation, etc... It seemed fine to just not bother testing twice, and only really test the part that we do (which is setting the attributes). It could be more detailed though since we do have that operationOptions assemblage happening in `createEventHubSpan`....